atlp-rwanda/trojans-ec-fe
# #184760057 - Add and view cart

### What does this PR do?

Adds the adding to  cart ,viewing it and clearing the whole cart both ui and functionalities

### Description of Task to be completed?
This is part of the cart operations done by the buyer 

### How should this be manually tested?

Mainly on the deployed site and npm test
PS: You have to be logged in to see the products
One of the user one can use testBuyer@example.com with password of default

The preview is available done below


### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/n/projects/2628461/stories/184760057

### Screenshots (if appropriate)
### Figma
#### Cart view
![figma](https://user-images.githubusercontent.com/111729060/233549064-1188a534-1b8b-4b16-90db-9a8be6501098.PNG)


### Actual
#### Cart view
![cart](https://user-images.githubusercontent.com/111729060/233548662-a2345406-90d4-4acb-9957-514ba92d11f9.PNG)



### Questions:

# Type of change

Please check the options that are relevant.


# Checklist:
